pkgs/by-name/sp/spl/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  pkgs,
+}:
+let
+  version = "0.3.1";
+  src = pkgs.fetchgit {
+    url = "https://git.tudbut.de/tudbut/spl";
+    rev = "v${version}";
+    hash = "sha256-Bj3voUwyqUWg1f86l/Vob+kb5jJUZmf0ZG9IxyYjgh0=";
+  };
+in
+pkgs.rustPlatform.buildRustPackage {
+  pname = "spl";
+  version = version;
+
+  src = src;
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
+  };
```

Comment:
```suggestion
```
This is an [import from derivation](https://nix.dev/manual/nix/2.24/language/import-from-derivation), which is not allowed in nixpkgs. `cargoHash` should be used if possible.